2-alcance/README.md

```diff
@@ -1,3 +1,187 @@
+
+- **Gestor de contenido:**
+  * **Agregar contenido**
+  * **Editar contenido**
+  * **Eliminar contenido**
+
+
+- **Gestor de los perfiles registrados:**
+  * **Gestión de perfiles:** el administrador, en el caso que sea necesario, podrá cambiar las funcionalidades concretas del usuario voluntario o incluso añadir nuevas.
+  
+  * **Gestión de la base de datos:** el administrado del sitio web tiene acceso a la base de datos de los perfiles registrados. 
+
+  * **Gestión del contenido adjuntado por el usuario voluntario:** el administrador será quien analice el contenido compartido por los usuarios voluntarios y también será quien decida qué contenido se publicará y cuál no. 
+
+  * **Gestión de incidencias:** teniendo en cuenta que el administrador tiene acceso a la base de datos de los perfiles resgistrados, este gestionará posibles incidencias que ocurran durante el funcionamiento del sitio web (dar de baja usuarios, eliminar contenido, etc.).
+
+
+- **Gestor de la configuración del sitio web:**
+  * El **resto de tareas** relacionadas con el sitio web serán responsabilidad del usuario administrador: configuración de contenido, diseño, etc. 
+
+## REQUERIMIENTOS DE CONTENIDOS
+
+### Contenido
+El contenido se divide en diferentes etapas:
+
+#### Audiencia
+Tras analizar las ventajas y desventajas de cada uno de los perfiles analizados anteriormente, es acertado considerar que adaptar la página web a una usuaria, __Dolores__ en este caso, que no tiene amplios conocimiento en lo que a navegación en la web se refiere será beneficioso para la accesibilidad del sitio web, puesto que aquellos usuarios avanzados en el mundo de Internet no tomarán por negativo un factor como tal, el que la interfaz y la navegación a través de la web sea sencilla, mientras que aquellos usuarios, como en este caso Dolores, que quieran disfrutar del contenido pero que no dispongan de un amplio conocimiento a la hora de navegar por la web, agradecerán que el sitio web sea accesible, lo que supondrá una ventaja, teniendo en cuenta que la ventaja competitiva reside en el contenido que se publicará en la web, y no tanto en el diseño o en la navegación.  
+
+#### Mensaje
+Dentro del sitio web, en el espacio dedicado a la publicación de los contenidos, el primer mensaje estará visible acompañada sobre una imagen descriptiva. El primer mensaje funcionará como hipervínculo en este caso. El segundo mensaje será el *lead* en el que se mostrará el resumen del texto, con las claves sobre las que se desarrollará el mismo. Viene a ser algo así como el resumen del contenido que se va a tratar en adelante. En caso de que los contenidos sean de otro tipo (vídeos o audios) será la descripción del mismo contenido. Se utilizarán palabras clave a modo de cumplir con las reglas *SEO* pero estas reglas no condicionarán el contenido del mismo.
```

Comment:
Aquí no se trata tanto de definir **dónde** irá el primer mensaje o el segundo mensaje, sino **cuáles son** exactamente esos mensajes, con qué palabras precisas los hacéis concretos.